CHANGELOG.txt

```diff
@@ -128,6 +137,15 @@
                 * New GraphRelationChartBrick that renders the relation graphs
            - Polls :
                 * The 'Statistic' brick does not use jQplot anymore.
+           - Reports:
+                * Add "creme.sketch" as dependency
+                * Add the following views to store the graph states in D3 bricks.
+                   - reports.views.graph.GraphFetchSettings
+                   - reports.views.graph.GraphFetchSettingsForInstance
+                * Add new brick for graph details view
+                   - ReportGraphChartBrick
+                * New USE_JQPLOT setting that switches from JQplot to D3 report graphs.
+                * The templatetag 'reports_graph_ordinate' is now deprecated; Use 'models.ReportGraph.verbose_ordinate' instead
```

Comment:
(Yes I know my Changelog is complex) there is a "Deprecations/apps" section for this item.